phpunit.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is there a convenient way to integrate this with Github and show code coverage there?